Battleships/Board Tests/GameManagerUnitTests.cs

```diff
@@ -39,11 +39,52 @@ public void Oyunu_Olusturup_Kullanicidan_Koordinat_Ister()
             var boardRenderer = new BoardRenderer(10, 10);
             IConsole console = new SystemConsole();
             List<Target> targets = new List<Target>();
+            Target amiralGemisi = new Target(4, Direction.East, "amiralGemisi");
+            Target kruvazor = new Target(3, Direction.North, "kruvazor");
+            Target mayinGemisi = new Target(2, Direction.East, "mayinGemisi");
+            Target denizalti = new Target(1, Direction.North, "denizalti");
+            targets.Add(amiralGemisi);
+            targets.Add(kruvazor);
+            targets.Add(mayinGemisi);
+            targets.Add(denizalti);
 
             var game = new Battleships.GameManager(console, playerBoard, computerBoard, boardRenderer, targets);
 
             game.Initialize();
-            game.Play();
+
+            game.UpdateGame("1,1");
+            game.UpdateGame("1,1");
+            game.UpdateGame("1,1");
```

Comment:
Bu kadar tekrar doğru gelmedi. Ne için bu?

Reply:
kullanıcıya 30 hak veriyorum oyun başında tüm hakları bitirmek için sürekli 1,1 koordinatı deneniyor gibi olsun diye böyle yapmıştım